apps/nft-squeeze/buildimgs.ts

```diff
@@ -0,0 +1,22 @@
+import * as glob from 'glob';
+import * as sharp from 'sharp'
+
+async function convertFiles() {
+  const files = glob.sync("./src/images/*.svg");
```

Comment:
This is an experiment - the build step converts from SVG into png/webp.  It is something we may consider for the other sites to improve load times